packages/count/src/index.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * @description This is a sample description
+ * @param {String} text
+ * @param (Function) validator, validates if a character should be count or not
+ * @example
+ * count('b') // returns 1
+ * count('Foo') // returns 3
+ * count('Foo bar') // returns 7
+ * count('Foo bar  ', char => char != ' ') //returns 6
+ * 
+ */
```

Comment:
I would be suggesting to exit early on the method or even using a default method as the validator.
```
const count = (text, validator = () => true) => {}
```

Reply:
I like validator = () => true) as default method